src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (x) => {
+    if (!isFinite(x)) {
```

Comment:
Nice variant. But also you can use equations
```suggestion
    if (!x && x !== 0) {
```